ykcapi/yk.h

```diff
@@ -71,9 +72,12 @@ __attribute__((naked)) void __llvm_deoptimize(void* addr, uintptr_t size) {
         "push rcx;\n"
         "push rdx;\n"
         "push rax;\n"
+        // 1st arg to yk_stopgap: stackmap addr (already in rdi).
+        // 2nd arg to yk_stopgap: stackmap size (already in rsi).
+        // 3rd arg to yk_stopgap: return address
         "mov rdx, [rsp+64];\n"
```

Comment:
Silly question: why `+64` if the 3 things mentioned above only add up to 24?

Reply:
The offset corresponds with the push instructions not the arguments. The 64 looks up the element on the stack when we enter the `llvm_deoptimize` call which is the return address.

Reply:
OK, then I'm not sure I understand why the comments are placed where they are? Can we reorg this a bit, and maybe explain the `64` too?

Reply:
The `1st` and `2nd` don't need to be there, but I don't know where else to put them. The `3rd` comment is there to tell us that `rdx` contains the return address which is an argument into `yk_stopgap`.

Reply:
I have no idea, other than to say that the current situation seems confusing :) @vext01 might have ideas?

Reply:
Try d8487c8